app/javascript/retrospring/features/answerbox/share.ts

```diff
@@ -0,0 +1,11 @@
+export function createShareEvent(answerbox: HTMLElement): () => void {
+  return function (): void {
+    navigator.share({
+      url: answerbox.querySelector<HTMLAnchorElement>('.answerbox__answer-date > a').href
+    }).then(() => {
+      // do nothing, prevents exce ption from being thrown
+    }).catch(() => {
+      // do nothing, prevents exception from being thrown
+    })
```

Comment:
Hmm, this looks kinda ugly, we could just do an empty function as a base `noop` instead.

Maybe create one in `retrospring/utilities`

```suggestion
    })
    .then(noop)
    .catch(noop)
```